.kokoro/build.sh

```diff
@@ -29,6 +29,9 @@ export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
 # Setup project id.
 export PROJECT_ID=$(cat "${KOKORO_GFILE_DIR}/project-id.json")
 
+# Set up system tests to run on different instances.
+export GOOGLE_CLOUD_TESTS_CREATE_SPANNER_INSTANCE=true
```

Comment:
Does this mean that we will create new spanner instances for system tests instead of using the existing one? The comment is not quite consistent with the variable name. The name is "create spanner instance" and the comment is "run on different instances". Could you please elaborate the comment a bit?

Reply:
I have updated the comment to match more closely to the variable name. Does the new comment clarify things?